connector_importer_source_sftp/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Camptocamp SA
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Connector Importer Source SFTP",
+    "summary": """Add import source capable of loading files from SFTP.""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Camptocamp,ACSONE,Odoo Community Association (OCA)",
+    "maintainers": ["simahawk", "sebalix"],
+    "website": "https://github.com/OCA/connector-interfaces",
+    "depends": [
+        "connector_importer",
+        "component_event",
+        "server_environment",
```

Comment:
Is it OK to enforce the use of this dependency to manage the configuration? Not better to use a new module for that?

Reply:
no, you are right, I trash it